client/auth/emailed-login-link-successfully.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import emailValidator from 'email-validator';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import { getCurrentQueryArguments } from 'state/ui/selectors';
+
+class EmailedLoginLinkSuccessfully extends React.Component {
+	render() {
+		const { translate, emailAddress } = this.props;
+		const line = [
+			emailAddress
+				? translate( 'We sent an email to %(emailAddress)s with a magic login link.', {
+					args: {
+						emailAddress
+					}
+				} )
+				: translate( 'We sent you an email with a magic login link.' ),
+			( <br key="magic-login-line-br" /> ),
+			translate( 'It should arrive within a few minutes. Go click it!', {
+				context: '"It" is an email'
+			} )
+		];
+
+		return (
+			<EmptyContent
+				title={ translate( 'Check your Email!' ) }
+				line={ line }
+				action={ translate( 'Back to WordPress.com' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Coming back to WordPress.com?' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).